PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -1683,23 +1713,23 @@ function Get-SddcDiagnosticInfo
 
             Show-Update "Start gather of cluster configuration ..."
 
-            $JobStatic += Start-Job -InitializationScript $CommonFunc -Name ClusterGroup {
+            $JobStatic += Invoke-SddcCommonCommand -ClusterNodes $AccessNode -JobName ClusterGroup -InitBlock $CommonFunc {

```

Comment:
Ah, here are at least a few of your problems.

Notice that not all of the start-job cases (in fact, most of them don't) require the CommonFunc. The Invoke-SddcCommonCommand pattern has a cost in the persistent powershell session that isn't torn down until much later. Thats actual cost - its a full instance of powershell.exe. Its not a cheap thread.

What you *could* do is avoid the persistent session if InitBlock isn't passed in, and that overhead goes away. Simply Invoke-Command -AsJob. But that becomes equivalent *if* ...

The real problem is that you converted all of the local jobs run by start-job to remoted jobs to the AccessNode. Thats not strictly a good idea, unless we needed to do that. Being able to run gathers using the local node's resources is a good thing, especially if it is outside the cluster. It avoids putting additional overhead in there.

It *also* means you aren't getting these gathers, since the $Path is a local path on the gathering node. At best there are files getting leaked around the cluster, unless I missed something else you tried. You can't have the remoted sessions write back to the gathering node since that would violate the double-hop restrictions.